css/properties.json

```diff
@@ -1720,6 +1720,22 @@
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
+  "alignment-baseline": {
+    "syntax": "baseline | alphabetic | ideographic | middle | central | mathematical | auto | before-edge | after-edge | text-before-edge | text-after-edge | hanging",
```

Comment:
I think the syntax should only include values in the current spec. We shouldn't list deprecated keywords. The good thing is that the content page already has a note to capture the deprecated list.
As far as BCD is concerned, we can determine if an update is needed.